security-privacy.html

```diff
@@ -194,11 +194,11 @@
     In order to support security policies concerning the Web NFC API, certain
     mechanisms need to be supported in all the implementations, such as:
     <ul>
-      <li>a mechanism for telling apart “Web NFC tags” from generic NFC tags
-        </li>
-      <li>a mechanism for white-listing URL's that can read, or write a certain
+      <li>an identification mechanism for telling apart “Web NFC tags” from
+        generic NFC tags</li>
+      <li>a mechanism for whitelisting URL's that can read, or write a certain
         Web NFC tag</li>
-      <li>a mechanism for black-listing URI's that can read, or write a certain
+      <li>a mechanism for blacklisting URI's that can read, or write a certain
```

Comment:
This sounds weird... read (past) write (present), "that can be read or written using..." ?